lms/djangoapps/student_account/test/test_views.py

```diff
@@ -243,6 +272,15 @@ def _change_password(self, email=None):
 
         return self.client.post(path=reverse('password_change_request'), data=data)
 
+    def _change_password_through_api(self, email=None):
```

Comment:
Since this is so closely tied to the password recover API (indeed, that's where it points), maybe we can rename this to `_recover_password_through_api` or something.